calico/reference/architecture/design/l3-interconnect-fabric.md

```diff
@@ -4,264 +4,130 @@ description: Understand considerations for implementing interconnect fabrics wit
-there is interest in the first approach, please contact Project Calico,
-and we can discuss, and if there is enough interest, maybe we will do
-another technical note on that approach. If you know of other approaches
-in use, we would be happy to host a guest technical note.
-
-1.  The routing infrastructure is based on some form of IGP. Due to the
-    limitations in scale of IGP networks (see the [why BGP
-    post](https://www.projectcalico.org/why-bgp/){:target="_blank"} for discussion of this
-    topic), the Project Calico team does not believe that using an IGP
-    to distribute endpoint reachability information will adequately
-    scale in a {{site.prodname}} environment. However, it is possible to use a
-    combination of IGP and BGP in the interconnect fabric, where an IGP
-    communicates the path to the *next-hop* router (in {{site.prodname}}, this is
-    often the destination compute server) and BGP is used to distribute
-    the actual next-hop for a given endpoint. This is a valid model,
-    and, in fact is the most common approach in a widely distributed IP
-    network (say a carrier's backbone network). The design of these
-    networks is somewhat complex though, and will not be addressed
-    further in this technical note. [^3]
-2.  The other model, and the one that this note concerns itself with, is
-    one where the routing infrastructure is based entirely on BGP. In
-    this model, the IP network is "tight enough" or has a small enough
-    diameter that BGP can be used to distribute endpoint routes, and the
-    paths to the next-hops for those routes is known to all of the
-    routers in the network (in a {{site.prodname}} network this includes the
-    compute servers). This is the network model that this note
-    will address.
-
-### BGP-only interconnect fabrics
-
-There are multiple methods to build a BGP-only interconnect fabric. We
-will focus on three models, each with two widely viable variations.
-There are other options, and we will briefly touch on why we didn't
-include some of them in the [Other Options appendix](#other-options).
+In a {{site.prodname}} network, each compute server acts as a router for all of the endpoints that are hosted on that compute server. We call that function a vRouter. The data path is provided by the Linux kernel, the control
+plane by a BGP protocol server, and management plane by {{site.prodname}}'s on-server agent, *Felix*.
+
+Each endpoint can only communicate through its local vRouter, and the first and last *hop* in any {{site.prodname}} packet flow is an IP router hop through a vRouter. Each vRouter announces all of the endpoints it is attached to to all the other vRouters and other routers on the infrastructure fabric, using BGP, usually with BGP route reflectors to
+increase scale. A discussion of why we use BGP can be found in {% include open-new-window.html text='Why
+BGP?' url='https://www.projectcalico.org/why-bgp/' %}.
```

Comment:
This is a link to a blog post on Tigera website. Good catch because it is an old link that resolves to: https://www.tigera.io/blog/why-bgp/. Updated the link.